pkg/operator/controller/status/controller_test.go

```diff
@@ -590,3 +590,67 @@ func TestComputeOperatorUpgradeableCondition(t *testing.T) {
 		}
 	}
 }
+
+func TestComputeOperatorEvaluationConditionsDetectedCondition(t *testing.T) {
```

Comment:
I can change the name, but then it will be inconsistent with the rest of the test names like `TestComputeOperatorUpgradeableCondition` or `TestComputeOperatorProgressingCondition`. So, I'll leave the decision to @Miciah.

Reply:
We recently had a discussion about this in our team channel.  Yes, Chad is right, we now prefer the style of `Test_computeOperatorEvaluationConditionsDetectedCondition` going forward.  (This is a minor thing though; I haven't been holding up PRs on this point.)

Reply:
Alright, I've updated the name.